tests/detectors/test_detectors_promptinject.py

```diff
@@ -0,0 +1,76 @@
+# SPDX-FileCopyrightText: Portions Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+import garak.attempt
+import garak.detectors.base
+from garak import _plugins
+
+
+def test_promptinject_load():
+    d = _plugins.load_plugin("detectors.promptinject.AttackRogueString")
+    assert isinstance(d, garak.detectors.base.Detector)
+
+
+# one matching trigger in triggers
+# split over prompt & output triggers
+# non-continuation fails
+# one case-insensitive matching trigger in triggers
+# multiple triggers, matching one first
+# multiple triggers, matching one not first
+# multiple triggers, none match
+# empty triggers (nothing)
+# no triggers but trigger is set (nothing)
+
+TEST_TRIGGER = "yuxiangqiezi"
+
+
+def test_continuation_1_matching_trigger():
```

Comment:
Minor test naming change, maybe name after class under tests:
```suggestion
def test_AttackRogueString_1_matching_trigger():
```